### PR TITLE
Update lanoising.py

### PR DESCRIPTION
**Pull request by Jibanandamht**
done bro

### CODE REVIEW

**Review by Jibanandamht — COMMENTED** [review]
@probhatmht check

**Comment by Probhatmht** [comment]
checked

File: scripts/lanoising.py [hunk 1]
```diff
@@ -531,3 +531,888 @@ def callback(msg):
     pub = rospy.Publisher('filtered_points', PointCloud2, queue_size=10)
 
     rospy.spin()
+        """PyTorch Hub models
+
+Usage:
+    import torch
```
Comment (Jibanandamht) [inline scripts/lanoising.py:537]: @Probhatmht

**Review by Probhatmht — CHANGES_REQUESTED** [review]
@Jibanandamht

**Review by Probhatmht — CHANGES_REQUESTED** [review]
@Jibanandamht

File: scripts/lanoising.py (hunk 1 above)
Comment (Jibanandamht) [reply scripts/lanoising.py:537]: done

**Review by Jibanandamht — COMMENTED** [review]
ok done

**Pull request closed without merging**